go/interpreter/lfvm/interpreter.go

```diff
@@ -143,26 +145,26 @@ func generateResult(ctxt *context) (tosca.Result, error) {
 
 	// Handle return status
 	switch ctxt.status {
-	case STOPPED, SUICIDED:
+	case statusStopped, statusSuicided:
 		return tosca.Result{
 			Success:   true,
 			GasLeft:   ctxt.gas,
 			GasRefund: ctxt.refund,
 		}, nil
-	case RETURNED:
+	case statusReturned:
 		return tosca.Result{
 			Success:   true,
 			Output:    res,
 			GasLeft:   ctxt.gas,
 			GasRefund: ctxt.refund,
 		}, nil
-	case REVERTED:
+	case statusReverted:
 		return tosca.Result{
 			Success: false,
 			Output:  res,
 			GasLeft: ctxt.gas,
 		}, nil
-	case INVALID_INSTRUCTION, OUT_OF_GAS, MAX_INIT_CODE_SIZE_EXCEEDED, ERROR:
+	case statusInvalidInstruction, statusOutOfGas, statusMaximumInitCodeSizeExceeded, statusError: // < TODO: if all these are handled the same, no need to have anything but statusError
```

Comment:
considering that throughout the interpreter these errors are treated differently and they are only considered equal in this point. Maybe if there was an if before the switch converting all theses status into `statusError` this could make sense.

Reply:
As I understand, they are not really "treated" differently, they are just "produced" differently. But this code is probably the only consumer, and it does not differentiate between those. So there is no point to have them differentiated.

We need to check whether there are places where those are actually used.